Makefile

```diff
@@ -15,7 +15,7 @@ e2e-cross: bin/$(BIN_NAME)-e2e-linux bin/$(BIN_NAME)-e2e-darwin bin/$(BIN_NAME)-
 
 .PHONY: bin/$(BIN_NAME)-e2e-windows
 bin/$(BIN_NAME)-e2e-%.exe bin/$(BIN_NAME)-e2e-%: e2e bin/$(BIN_NAME)-%
-	GOOS=$* $(GO_TEST) -c -o $@ ./$<
```

Comment:
why ? :thinking: (except it's probably more readable)

Reply:
Yep exactly for that reason 😄